Formula/cataclysm.rb

```diff
@@ -17,7 +17,6 @@ class Cataclysm < Formula
   depends_on "gettext"
   depends_on "libogg"
   depends_on "libvorbis"
-  depends_on "lua"
```

Comment:
If `lua` is still supported for stable, please leave it and only remove it for `head`.

Reply:
Is it possible to have different `make` arguments and `libexec.install` string for stable and experimental versions?

Lua-support depends on `LUA=1` and `lua` folder with some scripts which should be copied to folder with compiled exectuable.

Reply:
As far as I can tell, this brew formula doesn't support building stable from source, only the current HEAD of master.  Will add back the lua dependency and remove it for `head` once I figure out the syntax.